woodwork/tests/accessor/test_column_accessor.py

```diff
@@ -965,3 +965,18 @@ def test_nullable_attribute(sample_df_pandas):
         expected = EXPECTED_COLUMN_NULLABILITIES[key]
 
         assert actual is expected
+
+
+def test_validate(sample_df_pandas):
```

Comment:
We'll need special handling. When returning invalid values, we can return the dataframe and let users decide whether or not to compute or write to disk. Otherwise, we can just compute the scalar boolean value before raising an error if necessary.